examples/auction/src/lib.rs

```diff
@@ -275,46 +260,47 @@ mod tests {
     /// Test a sequence of bids and finalizations:
```

Comment:
IMO this unit test should be split into multiple or at least provide an error description to `claim_eq!`. 
If this unit test fails no useful error message is given.

Reply:
I provided some error descriptions to the `claim_eq!` now. Let me know if you think I should divide the tests further into smaller pieces.

Reply:
I am fine with this